.travis.yml

```diff
@@ -2,28 +2,12 @@ bundler_args: --without development
 language: ruby
 before_install: gem install bundler -v '<2'
 rvm:
-  - 1.9.3
-  - 2.0
-  - 2.1
-  - 2.2
   - 2.3
   - 2.4
   - 2.5.0
   - 2.6.3
-  - jruby-head
-  - rbx-2
-gemfile:
-  - Gemfile
-  - Gemfile-0.9.rb
+gemfile: Gemfile
```

Comment:
This should be useless as it's the default value for `gemfile`

Reply:
Cool, even easier to read!